src/Engine/Node.cpp

```
@@ -138,60 +147,96 @@ void node::deinit()
-    object_space_transform = mat4_to_array(ns->object_space_transform);
-    normal_transform = mat4_to_array(ns->normal_transform);
-    for (node* n : children)
+    ns->world_space_scale = new_world_space_scale;
+}
+
+void node::set_world_space_yaw(const float new_world_space_yaw) const
+{
+    ns->world_space_yaw = new_world_space_yaw;
+}
+
+void node::set_object_space_bounds(node_bounds new_object_space_bounds) const
```

Cppcheck (2.10-2)

[performance] Function parameter 'new_object_space_bounds' should be passed by const reference.

(passedByValue)